MACOSX_NOTES.md

```diff
@@ -150,27 +158,44 @@ Add rbenv to zsh/bash so that it loads every time you open a terminal
    rbenv global $RUBY_VERSION
 ```
 
-### Install an MO database snapshot.
+### Load an MO database snapshot.
 - download the snapshot from http://images.mushroomobserver.org/checkpoint_stripped.gz
-- copy (or move) the downloaded .gz file to the mushroom-observer directory
-Then
+- copy (or move) the downloaded .gz file to the mushroom-observer directory.
+Then:
 ```sh
 rake db:drop
 mysql -u root -p < db/initialize.sql
-Enter password:
 ```
-Enter `root` return
+When prompted to `Enter password:`<br>
+Enter `root`, return.
+Then:
 ```sh
 gunzip -c checkpoint_stripped.gz | mysql -u mo -pmo mo_development
 rails lang:update
 rails db:migrate
 ```
+When @JoeCohen first installed the app (MacBook Pro, Intel, OSX 13.6),
+`rails` was not recognized. He had to use `bin/rails` instead.
+The next time he installed the app (same hw/sw) ir recognized `rails`,
+but gave this error:
+```sh
+% rails lang:update
+rails aborted!
+Cannot load database configuration:
+Could not load database configuration. No such file - ["config/database.yml"]
+```
+which was fixed by running
+```sh
+cp db/macos/database.yml config
+```
+
 Then delete `checkpoint_stripped.gz` from the mushroom-observer directory
 
 ### Run the rest of the mo-dev script
 (See https://github.com/MushroomObserver/developer-startup/blob/main/mo-dev)
 (Both @nimmolo and @JoeCohen did this in pieces.)
-Open a new shell and run
+- Open a new shell
```

Comment:
@mo-nathan @nimmolo:
Why open a new shell here?

Reply:
No idea

Reply:
Worth noting that you also don't HAVE to delete checkpoint_stripped.gz anymore. 

It's in `.gitignore` and if you want to reload a db without downloading... there it is?